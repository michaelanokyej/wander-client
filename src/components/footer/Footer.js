import React from 'react'

class Footer extends React.Component {
  render () {
    return (
      <footer >Â© Michael Anokye - View <a href="https://github.com/michaelanokyej" >projects</a></footer>
    );
  }
}

export default Footer;