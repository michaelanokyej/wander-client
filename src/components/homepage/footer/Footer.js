import React from 'react'

class Footer extends React.Component {
  render () {
    return (
      <footer >Footer <a href="helpdesk-link">Help?</a></footer>
    );
  }
}

export default Footer;