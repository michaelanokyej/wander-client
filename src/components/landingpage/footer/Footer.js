import React from 'react'

class Footer extends React.Component {
  render () {
    return (
      <footer >Â© Michael Anokye - More <a href="https://github.com/michaelanokyej">projects</a></footer>
    );
  }
}

export default Footer;