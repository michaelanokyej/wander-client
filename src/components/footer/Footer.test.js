import React from "react";
import Footer from "./Footer"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

describe(`Footer component`, () => {
    it(`renders the Footer component`, () => {
        const wrapper = shallow(<Footer />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});