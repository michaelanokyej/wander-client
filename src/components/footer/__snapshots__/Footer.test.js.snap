// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the Footer component 1`] = `
<footer>
  Â© Michael Anokye - View 
  <a
    href="https://github.com/michaelanokyej"
  >
    projects
  </a>
</footer>
`;
